aplib.net-demo/Assets/Testing/PlaymodeTests/RangedEnemyTest.cs

```diff
@@ -0,0 +1,54 @@
+using Entities.Weapons;
+using NUnit.Framework;
+using System.Collections;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+using UnityEngine.TestTools;
+public class RangedEnemyTest
```

Comment:
Tests look good, but I'm a bit confused why this is called RangedEnemyTest. We're testing the ranged weapon (more specifically the ammo pouch) here right?